doc/user/logging.md

```diff
@@ -60,6 +60,53 @@ $ go run main.go --zap-level=1
 {"level":"info","ts":1559866310.065119,"logger":"scoped","msg":"Printing at INFO level"}
 {"level":"debug","ts":1559866310.065123,"logger":"scoped","msg":"Printing at DEBUG level"}
 ```
+## Custom zap logger
+
+In order to use a custom zap logger, [`zap`][controller_runtime_zap] from controller-runtime can be utilized to wrap it in a logr implementation.
+
+Below is an example illustrating the use of [`zap-logfmt`][logfmt_repo] in logging.
+
+### Example
+
+```Go
+package main
+
+import (
+	"github.com/go-logr/zapr"
+	zaplogfmt "github.com/sykesm/zap-logfmt"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	zapcr "sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var log logr.Logger
+
+func main() {
+	configLog := zap.NewProductionEncoderConfig()
+	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
```

Comment:
IHMO It could be something such as how to:

**Example**

In your main.go replace the following implementation;

```go
func main() {
        ....
	// Add the zap logger flag set to the CLI. The flag set must
	// be added before calling pflag.Parse().
	pflag.CommandLine.AddFlagSet(zap.FlagSet())

	// Add flags registered by imported packages (e.g. glog and
	// controller-runtime)
	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)

	pflag.Parse()

	// Use a zap logr.Logger implementation. If none of the zap
	// flags are configured (or if the zap flag set is not being
	// used), this defaults to a production zap logger.
	//
	// The logger instantiated here can be changed to any logger
	// implementing the logr.Logger interface. This logger will
	// be propagated through the whole operator, generating
	// uniform and structured logs.
	logf.SetLogger(zap.Logger())
```

With:

```go
	configLog := zap.NewProductionEncoderConfig()
	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
		encoder.AppendString(ts.UTC().Format(time.RFC3339))
	}
	logfmtEncoder := zaplogfmt.NewEncoder(configLog)

	// Construct a new logr.logger.
	log = zapcr.New(zapcr.UseDevMode(true), zapcr.WriteTo(os.Stdout), zapcr.Encoder(logfmtEncoder))

	// Set the controller logger to log, which will
	// be propagated through the whole operator, generating
	// uniform and structured logs.
	logf.SetLogger(log)
```

And then, ensure that you are using the following imports:

```go 
imports(
         ...
	"go.uber.org/zap/zapcore"
	"go.uber.org/zap"
	logf "sigs.k8s.io/controller-runtime/pkg/log"
	zapcr "sigs.k8s.io/controller-runtime/pkg/log/zap"
        zaplogfmt "github.com/sykesm/zap-logfmt"
        ...
)
```

**NOTE:** You will need to add the module `"github.com/sykesm/zap-logfmt"` to your project. Run `go get -u github.com/sykesm/zap-logfmt`

To test you can, for example, add the following print in the main function: 

```go
log.Info("Printing at DEBUG LEVEL")
```

So, when you run your operator you will check the following result. 

```
$ operator-sdk run --local 
INFO[0000] Running the operator locally in namespace default. 
ts=2020-02-27T20:03:14Z level=info msg="Printing at DEBUG LEVEL"
ts=2020-02-27T20:03:14Z level=info msg="Operator Version: 0.0.1"
ts=2020-02-27T20:03:14Z level=info msg="Go Version: go1.13.3"
ts=2020-02-27T20:03:14Z level=info msg="Go OS/Arch: darwin/amd64"
ts=2020-02-27T20:03:14Z level=info msg="Version of operator-sdk: v0.15.0+git"

```

Reply:
However, note that I perform these changes and the level still info. Should not it be debug?
@hasbro17 @estroz

Reply:
I did the changes and the level is info. Is it correct? Should not be debug?